### PR TITLE
[Url] replace utm_source iphone with android (AIC-386)

### PR DESCRIPTION
**Pull request by raigex**

### CODE REVIEW

File: db/src/main/kotlin/edu/artic/db/models/ArticDataObject.kt [hunk 1]
```diff
@@ -24,5 +24,28 @@ data class ArticDataObject(
         @Json(name = "membership_url") val membershipUrl: String?,
         @Json(name = "website_url") val websiteUrl: String?,
         @Json(name = "tickets_url") val ticketsUrl: String?,
-        @PrimaryKey val id : Int = 0
-)
\ No newline at end of file
+        @PrimaryKey val id: Int = 0
+) {
+
+    val memberShipUrlAndroid: String
```
Comment (Cliabhach) [inline db/src/main/kotlin/edu/artic/db/models/ArticDataObject.kt]: `memberShip` -> `membership`

File: info/src/main/java/edu/artic/info/MuseumInformationViewModel.kt [hunk 2]
```diff
@@ -46,7 +46,7 @@ class MuseumInformationViewModel @Inject constructor(
 
         dataObjectDao.getDataObject()
                 .toObservable()
-                .filterFlatMap({ it.ticketsUrl != null }, { it.ticketsUrl!! })
+                .map { it.ticketsUrlAndroid }
```
Comment (Cliabhach) [inline info/src/main/java/edu/artic/info/MuseumInformationViewModel.kt]: Don't we have to filter out empty strings here?

Comment (raigex) [reply info/src/main/java/edu/artic/info/MuseumInformationViewModel.kt]: yes we do done.

File: db/src/main/kotlin/edu/artic/db/models/ArticDataObject.kt (hunk 1 above)
Comment (raigex) [reply db/src/main/kotlin/edu/artic/db/models/ArticDataObject.kt]: done

**Pull request merged**